Common/pom.xml

```diff
@@ -140,10 +140,9 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <groupId>org.reflections</groupId>
-            <artifactId>reflections</artifactId>
+            <groupId>net.oneandone.reflections8</groupId>
+            <artifactId>reflections8</artifactId>
             <version>${reflections.version}</version>
-            <scope>provided</scope>
```

Comment:
should be provided, here. The specific implementations deal with shading. The common module simply assumes the implementation has the library.